syft/pkg/cataloger/apkdb/package_test.go

```diff
@@ -10,21 +10,26 @@ import (
 	"github.com/anchore/packageurl-go"
 	"github.com/anchore/syft/syft/linux"
 	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
 )
 
 func Test_PackageURL(t *testing.T) {
 	tests := []struct {
 		name     string
-		metadata pkg.ApkMetadata
+		metadata parsedData
 		distro   linux.Release
 		expected string
 	}{
 		{
 			name: "non-alpine distro",
-			metadata: pkg.ApkMetadata{
-				Package:      "p",
-				Version:      "v",
-				Architecture: "a",
+			metadata: parsedData{
+				"",
+				source.Location{},
+				pkg.ApkMetadata{
+					Package:      "p",
+					Version:      "v",
+					Architecture: "a",
+				},
```

Comment:
```suggestion
			metadata: parsedData{
				License: "",
				LicenseLocation: source.Location{},
				ApkMetadata: pkg.ApkMetadata{
					Package:      "p",
					Version:      "v",
					Architecture: "a",
				},
```

General comment: try to keep the field names to help future developers making changes here (harder to understand what is what without the names).